src/test/resources/hotels/hotels6.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Hotels passwords,emails
+  Scenario: Verify error message for invalid data in SignUp form
+    When I launch hotels website
+    And I click sign in button
+    And I click on SignUp link
+    When I click on email box
```

Comment:
Above step is continuation of action steps, so it should be labelled as And. Pls correct all such occurrences.